languages/csharp-dotnet-core2/template/src/Algorithm.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Algorithmia;
+
+//This is an example of an algorithm, you can evaluate it with this input:
+/// Example Input:
+/// {"name": "Algorithmia user"}
+///
+/// Expected Output:
+/// {"output":"Hello Algorithmia user"}
+namespace Algo
+{
+    public class AlgoInput
+    {
+        
+     /// The input object type for your algorithm, this must contain json serializable types.
```

Comment:
Seems like this comment should be at the top of the class, not the field?

Reply:
Moved it, along with adding a bit more documentation in the latest commit.